presto-main/src/main/java/io/prestosql/connector/informationschema/InformationSchemaSplit.java

```diff
@@ -16,77 +16,39 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.common.collect.ImmutableList;
-import com.google.common.collect.ImmutableSet;
-import io.prestosql.metadata.QualifiedTablePrefix;
 import io.prestosql.spi.HostAddress;
 import io.prestosql.spi.connector.ConnectorSplit;
 
 import java.util.List;
-import java.util.Set;
 
-import static com.google.common.base.MoreObjects.toStringHelper;
-import static com.google.common.base.Preconditions.checkArgument;
 import static java.util.Objects.requireNonNull;
 
 public class InformationSchemaSplit
         implements ConnectorSplit
 {
-    private final InformationSchemaTableHandle tableHandle;
-    private final Set<QualifiedTablePrefix> prefixes;
-    private final List<HostAddress> addresses;
+    private final HostAddress address;
 
     @JsonCreator
-    public InformationSchemaSplit(
-            @JsonProperty("tableHandle") InformationSchemaTableHandle tableHandle,
-            @JsonProperty("prefixes") Set<QualifiedTablePrefix> prefixes,
-            @JsonProperty("addresses") List<HostAddress> addresses)
+    public InformationSchemaSplit(@JsonProperty("address") HostAddress address)
     {
-        this.tableHandle = requireNonNull(tableHandle, "tableHandle is null");
-        this.prefixes = ImmutableSet.copyOf(requireNonNull(prefixes, "prefixes is null"));
-
-        requireNonNull(addresses, "hosts is null");
-        checkArgument(!addresses.isEmpty(), "hosts is empty");
-        this.addresses = ImmutableList.copyOf(addresses);
+        this.address = requireNonNull(address, "address is null");
     }
 
     @Override
     public boolean isRemotelyAccessible()
     {
-        return false;
+        return true;
```

Comment:
Is this correct?  I thought information schema only ran on the coordinator.

Reply:
Good catch, this needs to be reverted